packages/marko-web-theme-monorail/scss/core.scss

```diff
@@ -0,0 +1,59 @@
+@import "functions";
+@import "variables";
+
+@import "../../node_modules/@parameter1/base-cms-marko-web-theme-default/scss/theme";
+@import "../../node_modules/bootstrap/scss/spinners";
+
+@import "../../node_modules/@parameter1/base-cms-marko-web-social-sharing/scss/buttons";
+@import "../../node_modules/bootstrap/scss/input-group";
+@import "../../node_modules/@parameter1/base-cms-marko-web-html-sitemap/scss/index";
+
```

Comment:
Naughty! Remove the default theme references entirely. If you need that theme file, re-create it in this package.

Note the bootstrap comment above, this is required for the input-group and spinners imports. Remove the import for input-group from the other file -- don't include it twice.

If you need to import the css for the sitemap and social sharing packages, make sure they are in your package.json as well.